### PR TITLE
[#2053] Add Aria To Export Dropdown

### LINKED ISSUE #2053: Dropdown Does Not Have Accessible Name
The [export dropdown](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ExportDropdown.js#L28) on record landing page does not have an accessible name. 
https://dequeuniversity.com/rules/axe/4.4/aria-input-field-name

To fix, add `aria-label` to the dropdown.

### PR DESCRIPTION
**Pull request by sharpattack**
The export dropdown needs an accessible name. fixes #2053

:heart: Thank you for your contribution!

### Description

Please describe briefly your pull request.

### Checklist

Ticks in all boxes and 🟢 on all GitHub actions status checks are required to merge:

- [x] I'm aware of the [code of conduct](https://inveniordm.docs.cern.ch/contribute/code-of-conduct/).
- [x] I've created [logical separate commits](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commits) and followed the [commit message format](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#commit-message).
- [ ] I've added relevant test cases.
- [ ] I've added relevant documentation.
- [ ] I've marked [translation strings](https://inveniordm.docs.cern.ch/develop/best-practices/i18n/) (for relevant code).
- [ ] I've followed the [CSS/JS](https://inveniordm.docs.cern.ch/develop/best-practices/css-js/) and [React](https://inveniordm.docs.cern.ch/develop/best-practices/react/) guidelines (for relevant code).
- [x] I've followed the [web accessibility](https://inveniordm.docs.cern.ch/develop/best-practices/accessibility/) guidelines (for relevant code).
- [ ] I've followed the [user interface](https://inveniordm.docs.cern.ch/develop/best-practices/ui/) guidelines (for relevant code).
- [x] I've identified the [copyright holder(s)](https://inveniordm.docs.cern.ch/contribute/copyright-policy/) and updated copyright headers for touched files (>15 lines contributions).
- [x] I've NOT included third-party code (copy/pasted source code or new dependencies).

**Third-party code**

If you've added [third-party code (copy/pasted or new dependencies)](https://inveniordm.docs.cern.ch/develop/best-practices/commits/#third-party-codedependencies), please reach out to an [architect](https://github.com/orgs/inveniosoftware/teams/architects/members).

**Reminder**

By using GitHub, you have already agreed to the [GitHub’s Terms of Service](https://help.github.com/articles/github-terms-of-service/#6-contributions-under-repository-license) including that:

1. You license your contribution under the same terms as the current repository’s license.
2. You agree that you have the right to license your contribution under the current repository’s license.

### CODE REVIEW

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ExportDropdown.js [hunk 1]
```diff
@@ -1,3 +1,9 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2023 Northwestern University.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
```
Comment (fenekku) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ExportDropdown.js:4]:
```suggestion
// Copyright (C) 2023 CERN.
// Copyright (C) 2023 Northwestern University.
//
// Invenio App RDM is free software; you can redistribute it and/or modify it
```

I know the other ones have "Invenio RDM Records" but that's probably just careless copy-paste.

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ExportDropdown.js [hunk 2]
```diff
@@ -26,6 +32,7 @@ export class ExportDropdown extends Component {
       <Grid>
         <Grid.Column width={11}>
           <Dropdown
+            aria-label="Export selection"
```
Comment (fenekku) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ExportDropdown.js:35]: We also probably want internationalization on the core module if we don't use it at NU:

```suggestion
            aria-label={i18next.t("Export selection")}
```

**Review by fenekku — APPROVED** [review]
Thanks for this. Good for me after minor comments are addressed.

**Review by jennur — APPROVED** [review]
Thanks! 🙏

**Review by fenekku — APPROVED** [review]
Thanks!

**Pull request merged**